### PR TITLE
makefile:chore - updating makefile to avoid outdate dependencies

### PR DESCRIPTION
**Pull request by nathanmartinszup**
Signed-off-by: Nathan Martins <nathan.martins@zup.com.br>

**- What I did**
This pull request changes the way that we install some tools that we use in our makefile, the ideia is avoid outdated dependencies and use the auto updates by renovate bot. Also added a format command with two news auto formaters [gofumpt](https://github.com/mvdan/gofumpt) and [gci](https://github.com/daixiang0/gci).

This pull request depends on the #809, which contains all the changes made by the formaters. 

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,20 +35,23 @@ test-e2e:
...
+	$(GO_GCI) -w -local $(GO_IMPORTS_LOCAL) $(GO_FILES)
+
+install-format-dependencies:
+	$(GO) install golang.org/x/tools/cmd/goimports@v0.1.7
+	$(GO) install mvdan.cc/gofumpt@v0.2.0
+	$(GO) install github.com/daixiang0/gci@v0.2.9
 
 license:
-	$(GO) install github.com/google/addlicense
+	$(GO) install github.com/google/addlicense@v1.0.0
```
Comment (matheusalcantarazup) [inline Makefile:50]: Can we use `@latest` here?

File: Makefile [hunk 2]
```diff
@@ -40,20 +35,23 @@ test-e2e:
...
 	@$(ADDLICENSE) -check -f ./copyright.txt $(shell find -regex '.*\.\(go\|js\|ts\|yml\|yaml\|sh\|dockerfile\)')
 
 license-fix:
-	$(GO) install github.com/google/addlicense
+	$(GO) install github.com/google/addlicense@v1.0.0
```
Comment (matheusalcantarazup) [inline Makefile:54]: And here?

File: Makefile [hunk 3]
```diff
@@ -40,20 +35,23 @@ test-e2e:
...
+format: install-format-dependencies
+	$(GOFMT) -s -l -w $(GO_FILES)
+	$(GO_IMPORTS) -w -local $(GO_IMPORTS_LOCAL) $(GO_FILES)
+	$(GO_FUMPT) -l -w $(GO_FILES)
+	$(GO_GCI) -w -local $(GO_IMPORTS_LOCAL) $(GO_FILES)
+
+install-format-dependencies:
+	$(GO) install golang.org/x/tools/cmd/goimports@v0.1.7
+	$(GO) install mvdan.cc/gofumpt@v0.2.0
+	$(GO) install github.com/daixiang0/gci@v0.2.9
```
Comment (matheusalcantarazup) [inline Makefile:47]: And here?

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (nathanmartinszup) [reply Makefile]: Done!

File: Makefile (hunk 3 above)
Comment (nathanmartinszup) [reply Makefile]: Done!

File: Makefile (hunk 2 above)
Comment (nathanmartinszup) [reply Makefile]: Done!

**Review by matheusalcantarazup — APPROVED** [review]

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**